stable/README.md

```diff
@@ -60,41 +60,29 @@ export HELM_HOME=`pwd`/.helm
 export KUBECONFIG=`pwd`/.kube/config
 ```
 
-### Create a Project
-
-We're limiting the Tiller server to manage infrastructure in the project (namespace) where it is installed. So the TILLER_NAMESPACE equals the namespace where NuoDB will be installed.
```

Comment:
There are two approaches that we normally use with vanilla K8s:
- Namespace scoped Tiller installation that should work for all charts by StorageClass
- System scoped Tiller installation that was used for StorageClass chart (see `test/scripts/prepare.sh`)

I look like we are removing the steps for namespace scoped tiller installation with these changes. Are we saying that we are going to support only system scoped Tiller installation?

Reply:
Is it relevant for the purposes of our repo - what we are giving is guidance on an option of how to set up helm, not supporting it?  In reality if a customer is using Helm they will likely already have it installed with the correct permissions, following some form of internal guidance... This will also go away when we go to 3.0

Reply:
For OpenShift I want to have Tiller running globally. The THP should also be running just once, not once for every project. So I changed the wording in this doc to reflect that. I basically made the opposite recommendation.